installing/installing_aws/installing-aws-outposts.adoc

```diff
@@ -0,0 +1,16 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="installing-aws-outposts"]
+= Installing a cluster on AWS with compute nodes on AWS Outposts
+include::_attributes/common-attributes.adoc[]
+:context: installing-aws-outposts
+
+toc::[]
+
+In {product-title} version 4.14, you could install a cluster on
+Amazon Web Services (AWS) with compute nodes running in AWS Outposts as a Technology Preview.
+As of {product-title} version {product-version}, this installation method is no longer supported.
+Instead, you can
+xref:../../installing/installing_aws/installing-aws-vpc.adoc#installing-aws-vpc[install a cluster on AWS into an existing VPC]
+and provision compute nodes on AWS Outposts as a postinstallation configuration task.
```

Comment:
```suggestion
and provision compute nodes on AWS Outposts as a post installation configuration task.
```

Reply:
I've got it as "postinstallation" because that's how we refer to it in the main doc: https://docs.openshift.com/container-platform/4.15/post_installation_configuration/index.html

<img width="1049" alt="Screenshot 2024-02-21 at 9 56 44 PM" src="https://github.com/openshift/openshift-docs/assets/93731574/8006305e-7694-4efb-a016-aa67fad734b5">

Reply:
I was wondering about it too. SGTM :)

Reply:
I don't think anyone likes it, but that's it the style guide :sweat_smile: